packages/web/src/djitsu/components/notebook-header/index.js

```diff
@@ -291,6 +297,57 @@ export const NotebookHeader = (props) => {
   //   { hasMain, notebook, userProfile, notebookId, isOwner }
   // )
 
+  const fallbackCopyTextToClipboard = (text) => {
```

Comment:
I'd extract this entire logic into a separate function or a custom hook - and i'd also check if there's a copy-to-clipboard package to use instead of hand-crafting it